assets/scss/components/_service-info.scss

```diff
@@ -195,3 +195,48 @@ Styleguide Service Info.Logged In
 .last-login__more-details {
   margin: em(5) 0 0 0;
 }
+
+/*
+Floated items
+
+Service Info with `.service-info__floated-items` allows for content to be floated on the left and right sides with adequate margins. On mobile breakpoints content is displayed block
+
+Markup:
+<div class="service-info ">
+  <div class="service-info__floated-items">
+    <div class="service-info__left-item">
+      <span>Left Side</span>
+    </div>
+    <div class="service-info__right-item">
+      <span>Right Side</span>
+    </div>
+  </div>
+</div>
+
+Styleguide Service Info.Left and Right Items
+*/
+.service-info__floated-items {
```

Comment:
But as it's currently not padding, it's a margin that's already being set through other selectors.

if all you're depending on a wrapper for is a margin top then you're better off getting rid of it and adding a `service-info__item` class to each child you want the margin applied to.

That class can then be added to the list of classes that already have the margin on so that refactoring is easier